### PR TITLE
[proposal] Changed MSSQL url to active project. Original has gone stale.

### PR DESCRIPTION
**Pull request by misterGF**
[proposal] 
Change the community MSSQL waterline adapter link to an active project. The one that is linked to works well but there hasn't been active development. The new link is a fork of the original project with some feature enhancements and proper testing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ Sails is built on [Node.js](http://nodejs.org/), [Connect](http://www.senchalabs
...
.../balderdashy/sails-mongo), [SQLite3](https://github.com/AndrewJo/sails-sqlite3), [Redis](https://github.com/balderdashy/sails-redis), local [disk](https://github.com/balderdashy/sails-disk), and local [memory](https://github.com/balderdashy/sails-memory).  [Community adapters](https://github.com/balderdashy/sails-docs/blob/master/contributing/intro-to-custom-adapters.md#notable-community-adapters) exist for [CouchDB](https://github.com/search?q=sails+couch&nwo=codeswarm%2Fsails-couchdb-orm&search_target=global&ref=cmdform), [neDB](https://github.com/adityamukho/sails-nedb), [TingoDB](https://github.com/andyhu/sails-tingo), [SQLite](https://github.com/AndrewJo/sails-sqlite3/tree/0.10), [Oracle](https://github.com/search?utf8=%E2%9C%93&q=%22sails+oracle%22+OR+%22waterline+oracle%22&type=Repositories&ref=searchresults), [MSSQL](https://github.com/misterGF/sails-mssqlserver), [DB2](https://github.com/search?q=sails+db2&type=Repositories&ref=searchresults), [ElasticSearch](https://github.com/search?q=%28elasticsearch+AND+sails%29+OR+%28elasticsearch+AND+waterline%29+&type=Repositories&ref=searchresults), [Riak](https://github.com/search?q=sails+riak&type=Repositories&ref=searchresults),
```
Comment (mikermcneil) [inline README.md]: (complete aside:  maaannnnn I've got to figure out a way to make this part of the README easier to look at in diffs...)

Comment (mikermcneil) [inline README.md]: for other folks landing on this PR, here's a link to the repo for easy reference: https://github.com/misterGF/sails-mssqlserver

**Comment by mikermcneil** [comment]
@misterGF thanks!

@felixrieseberg @sedouard do you guys have a go-to strategy you use for testing open-source stuff against MSSQL?  Right now, looks like the [circle ci tests](https://circleci.com/gh/misterGF/sails-mssqlserver/10#tests) need some help running [waterline-adapter-tests](http://github.com/balderdashy/waterline-adapter-tests):
image

**Comment by felixrieseberg** [comment]
AppVeyor is a neat CI engine that supports all kinds of SQL Server: https://www.appveyor.com/docs/services-databases

**Comment by misterGF** [comment]
All set. Thanks for the suggestion. https://ci.appveyor.com/project/misterGF/sails-mssqlserver

**Comment by odnarb** [comment]
This might be a discussion for misterGF's MSSQL repo, but I am concerned this new repo came from "nowhere". There's no real network map. Wasn't this forked from somewhere else? It would be really nice to maintain that history in its repo because I have a team that's trying to use the most relevant MSSQL adapter, which has been insane. 

This is the original as far as I can tell and here's the network map.
https://github.com/swelham/sails-mssql/network

Can you comment on this @misterGF ? Thanks

**Comment by misterGF** [comment]
Hi @odnarb. You are right. The project was originally forked from https://github.com/cnect/sails-sqlserver
I've been using it pretty heavily in my projects and made some changes that I think makes it better. Contributed back to the project but haven't had much luck with responses. That's why I decided to make it it's own thing and had github support make the project independent. I believe by doing that it removes the network map info. I'll send a follow up to them to see if there is a way to maintain that info.

**Comment by mikermcneil** [comment]
@felixrieseberg thanks!

@misterGF In case it saves you some time, I took a pass at a "Credits" section here: https://github.com/misterGF/sails-mssqlserver/pull/1   (cc @odnarb)

**Comment by misterGF** [comment]
Merged. Thank you very much!

**Comment by odnarb** [comment]
@misterGF Thanks for the response, that helps us understand the state of things. There we other forks that made good bugfixes. 

The most interesting forks in terms of bugfixes from the cnect repo look to be "catdad" and "YouShouldTeam".

Did you happen to look through those forks, their bugs and merge them in?

Thank you!

**Comment by misterGF** [comment]
I haven't yet but I will. Thanks for your input.

**Comment by odnarb** [comment]
Thanks!

**Pull request merged**